common/smoke-test/smoke-tests.yml

```diff
@@ -6,30 +6,43 @@ jobs:
           OSVmImage: "ubuntu-18.04"
           SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources)
           ArmTemplateParameters: $(AzureCloudArmTemplateParameters)
+          NodeTestVersion: "8.x"
```

Comment:
Out of curiosity how did you end up picking the NodeTestVersion? Is it random or did you follow some pattern to identify a sparse matrix?

Reply:
I based it off the table @mikeharder shared at https://github.com/Azure/azure-sdk-for-js/issues/9462. Not sure whether he had a heuristic or not.

Reply:
I created the test matrix manually, with a few goals:

- Test all OS, node versions, and browser versions at least once across all tests
- Test all node versions at least once across each test type (PR, live, smoke)

Given these constraints I manually created something which seems reasonable.  It wasn't an exact science.